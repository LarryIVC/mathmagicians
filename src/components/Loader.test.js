import React from 'react';
import { render } from '@testing-library/react';
import Loader from './Loader';
import '@testing-library/jest-dom/extend-expect';

test('renders Loader component', () => {
  const { container } = render(<Loader />);
  expect(container).toMatchSnapshot();
});
