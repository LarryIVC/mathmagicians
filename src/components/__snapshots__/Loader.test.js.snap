// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loader component 1`] = `
<div>
  <div
    class="lds-spinner"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
